cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -238,104 +238,75 @@ const updateHostedZoneRecords = async function (
 // deleteHostedZoneRecords deletes the validation records associated with the certificate.
 // We don't want to delete a validation record if it's used by another certificate because
 // the validation records are used to renew the certificate.
-// The legacy certificate (only `${envName}.${appName}.${domainName}`) and the new
-// certificate generated with the "alias" field share the same validation record.
-// Therefore, we check if there is more than one certificate using the record and only delete
-// if there is no other certificate using the record.
 const deleteHostedZoneRecords = async function (
-  options,
-  certDomain,
+  oldCertOptions,
+  certArn,
```

Comment:
```suggestion
  oldCertArn,
```
I think this may help understanding the rest of the code!